src/components/OSCALControlProse.js

```diff
@@ -328,143 +319,207 @@ export function OSCALReplacedProseWithParameterLabel(props) {
+            <Link href="#{props.label}">{props.label}</Link>
+          </StyledTooltip>
+          {proseDisplay}
+          {props.modificationDisplay}
+        </Typography>
+      </Grid>
+      <Grid item xs={1} className={classes.OSCALStatementEditControlsContainer}>
+        {(props.isEditable && !isEditingStatement) ? (
           <IconButton
+            size="small"
             onClick={(event) => {
-              setAnchorEl(event.currentTarget);
+              setIsEditingStatement(!isEditingStatement);
             }}
           >
             <EditIcon fontSize="small" />
           </IconButton>
-          <Grid container>
-            <OSCALControlPartEditor
-              anchorEl={anchorEl}
-              setAnchorEl={setAnchorEl}
-              controlId={props.controlId}
-              isUserDefinedImplementation={props.prose.match(
-                prosePlaceholderRegexpString
-              )}
-              onCancel={() => {
-                setAnchorEl(null);
-              }}
-              onFieldSave={(descriptionReference, implementationReference) => {
-                onFieldSaveByComponentParameterValue(
-                  props,
-                  descriptionReference,
-                  implementationReference
-                );
+        ) : null}
+      </Grid>
+      {isEditingStatement ? (
+        <>
+          <Grid item xs={10}>
+            <TextField
+              fullWidth
+              label="Description"
+              multiline
+              variant="outlined"
+              inputProps={{
+                "data-testid": "Statement By Component Description TextField",
               }}
-              statementByComponent={statementByComponent}
-              statementId={props.statementId}
+              inputRef={statementByComponentDescriptionRef}
+              defaultValue={statementByComponentDescription}
             />
           </Grid>
+          {!isProcessingRequest ? (
+            <>
+              <Grid item xs={1} className={classes.OSCALStatementEditControlsContainer}>
+                <IconButton
+                  aria-label={`save-${[props.statementId]}`}
+                  onClick={() => {
+                    restUtils.createOrUpdateSspControlImplementationImplementedRequirementStatementByComponent(
+                      props.partialRestData,
+                      props.implementedRequirement,
+                      props.statementId,
+                      props.componentId,
+                      statementByComponentDescriptionRef.current.value,
+                      Object.keys(setParametersRefs).map((setParameterId) => {
+                        const setParameterValue = setParametersRefs[setParameterId]?.current?.value;
+                        return (setParameterValue) ? {
+                          "param-id": setParameterId,
+                          values: setParameterValue
+                        }
+                        :
+                        null
```

Comment:
Is this the linter formatting like this? Can we somehow convince it to be okay with

```suggestion
                        return (setParameterValue)
                          ? {
                            "param-id": setParameterId,
                            values: setParameterValue
                          }
                          : null
```

This is a nitpick but having that null on it's own line took me a minute to parse what's going on here

Reply:
I hadn't run linting at all yet, just wanted to get people to start looking at the approach.  Should be all linted now.